java/code_to_gast/java_expression.py

```diff
@@ -12,9 +12,34 @@ def method_invocation_to_gast(node):
     if node.qualifier == "System.out" and node.member == "println":
         gast["value"] = {"type": "logStatement"}
     else:
-        gast["value"] = {"type": "customStatement"}
+        # function called on object
+        if node.qualifier:
+            object_list = (node.qualifier.split("."))
+            object_list.append(node.member)
+            gast["value"] = list_to_attribute_value_node(object_list)
+        else:
+            gast["value"] = {"type": "name", "value": node.member}
     return gast
 
+'''
+Takes list of callees and members and from function called
+on object and translates into gAST node
+Ex: car.drive() -> {"type": "attribute", "id": "drive", "value": 
+{"type": "name", "id": "car"}}
+'''
+def list_to_attribute_value_node(object_list):
+    gast = {}
+    gast["type"] = "attribute"
```

Comment:
Can we create a ticket to move all hardcoded strings like "attribute" to enums in the Java code? I would label these with a tag that says "Refactor"

Reply:
+1